packages/core-data/src/entity-provider.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext, useCallback } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { defaultEntities, kinds } from './entities';
+
+const _entities = {
```

Comment:
Can we avoid introducing new coding patterns for consistency (we don't really use `_something` in our codebase, I assume this is a way to tell it's a private thinig)